app/services/api_seed_data/declarations.rb

```diff
@@ -0,0 +1,175 @@
+module APISeedData
+  class Declarations < Base
+    MAX_TEACHERS_WITH_DECLARATIONS = 50
+
+    def plant
+      return unless plantable?
+
+      log_plant_info("declarations")
+
+      active_lead_providers.product(teacher_types).each do |active_lead_provider, teacher_type|
+        log_seed_info("#{active_lead_provider.lead_provider.name} - #{teacher_type}", indent: 4)
+
+        MAX_TEACHERS_WITH_DECLARATIONS.times do
+          school_partnership = find_random_school_partnership(active_lead_provider)
+          next unless school_partnership
+
+          training_periods = find_random_training_periods_without_declarations(school_partnership, teacher_type)
+          next if training_periods.blank?
+
+          create_declarations!(active_lead_provider, training_periods)
+        end
+      end
+    end
+
+  protected
+
+    def plantable?
+      super && Declaration.none?
```

Comment:
Hi @ethax-ross There's one declaration being created when running the normal seeds locally.. so this is not running.. Is there anyway we can run both if local?

Reply:
The two seeds are totally separate now, so to run locally you need to wipe your DB and then just run the API seeds; they don't depend on any of the dev seeds any more so can run from scratch